src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_STUDENTS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.student.Id;
+import seedu.address.model.student.Student;
+import seedu.address.model.student.Training;
+
+/**
+ * Edits the details of an existing student in the address book.
+ */
+public class DeleteStudentCommand extends Command {
+
+    public static final String COMMAND_WORD = "ts-delete";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes the corresponding Students "
+            + "from the specified Training Session"
+            + " PARAMETERS: TRAINING_SESSION-ID STUDENT_ID..."
+            + "\nExample: "
+            + COMMAND_WORD + " 1 3,5,7";
+
+    public static final String MESSAGE_ADD_STUDENT_SUCCESS = "Deleted Student: %1$s";
```

Comment:
I think should change the name of the variable to match the usage. To me it doesn't look nice ah.